url_shortener/cmd/httpload/main.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"sort"
+	"strings"
+)
+
+const (
+	exitCodeOk    = 0
+	exitCodeError = 1
+)
+
+func computeStatuses() (statuses map[int]int, err error) {
+	statuses = make(map[int]int)
+
+	statuses[200] = 198
+	statuses[404] = 2
+
+	return statuses, err
```

Comment:
`err` is never used!

Reply:
Nitpicking: `err` is used in `main` function.

At the moment, I only developed the print statistics part and I am still unsure that `computeStatuses` will be part of the final API.
However, if `computeStatuses` is going to be the final API it has to return errors in case of problems, so I defined the `err` variable. This has the nice side effect that the `main` function is already "ready for production".
Currently, `computeStatuses` has a fake implementation, so it cannot make "mistakes": this is the reason is always returning a nil `err` value. For the moment. :smile: